e2e/runtime_via_lima_test.go

```diff
@@ -0,0 +1,127 @@
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/arm/remoteproc-runtime/e2e/limavm"
+	"github.com/arm/remoteproc-runtime/e2e/remoteproc"
+	"github.com/arm/remoteproc-runtime/e2e/repo"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRuntimeProxyKeepsHostNamespaceWhenNotRootInLimaVM(t *testing.T) {
```

Comment:
```suggestion
func TestRuntimeProxyKeepsHostNamespaceWhenNotRoot(t *testing.T) {
```

LimaVM is not relevant. It should happen everywhere.